knowledge-base/troubleshoot-mac-proxy-settings.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Troubleshoot Mac Proxy Settings
+description: Troubleshooting Mac Proxy Settings will help isolate Issues
+type: troubleshooting
+page_title: Verifying Mac Proxy Settings
+slug: troubleshoot-mac-proxy-settings
+position: 
+tags: troubleshooting, apple, mac, proxy
+ticketid: 1388381
+res_type: kb
+category: knowledge-base
```

Comment:
remove category: knowledge-base

Reply:
Also removed this in the other KB article as well.